content/en/continuous_delivery/deployments/argocd.md

```diff
@@ -177,13 +177,17 @@ The [**Deployments**][6] and [**Executions**][7] pages populate with data after
 
 ## Correlate deployments with CI pipelines
 
-If Argo CD deployments are triggered from a CI pipeline, you can correlate the deployment execution and the pipeline. This allows you to visualize all the deployments triggered from a pipeline, as well as which pipeline triggered a specific deployment. See the [Visualize correlated pipelines and deployments][18] section on the CD Visibility page for an example.
+By default, the Git metadata reported in deployment events is associated with the repository that Argo CD monitors. However, a common setup is to:
+- Have an application repository, storing the source code, and a configuration repository, storing the Kubernetes manifests. Then, configure Argo CD to monitor the configuration repository, as outlined in the [Argo CD Best Practices page][17].
+- When a change occurs in the application repository, perform an automated commit that updates the configuration repository (for example, changing the current image of a Kubernetes resource).
 
-By default, the Git metadata reported in deployments is associated with the repository that Argo CD monitors. However, a common setup is to separate the source code repository from the repository Argo CD monitors (the configuration repository), as outlined in the [Argo CD Best Practices page][17]. Then, you can use automated commits from a CI pipeline to update the configuration repository when changes occur in the source repository. The following diagram represents an example of this kind of setup:
+The following diagram represents an example of this kind of setup:
 
 {{< img src="ci/cd-argocd-ci-correlation-setup-git.png" alt="Triggering Argo CD deployments using git" style="width:100%;">}}
 
-To associate source code Git information with the Argo CD deployments, run the `datadog-ci deployment correlate` command before pushing the changes to the configuration repository. See the [command syntax][14] for additional details:
+In this case, you can replace the Git metadata reported in the deployment with the one of the application repository instead of the configuration repository. This allows you to connect the deployments performed by Argo CD and the related CI pipeline runs on the application repository.
```

Comment:
```suggestion
In this case, you can replace the Git metadata reported in the deployment with the metadata of the application repository instead of the configuration repository. This allows you to connect the deployments performed by Argo CD and the related CI pipeline runs on the application repository.
```